cloud/aws/benchmarks/benchmarks.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# The caller should download the static-tests and extract them into the
+# local directory.
+
+set -x
+
+TEST_FLAGS="-test.bench=.* -test.benchmem -test.run=NONE -test.count=10"
```

Comment:
@petermattis will know whether this has a cost that would skew the actual performance.